deploy/helm-chart/values.yaml

```diff
@@ -45,6 +45,10 @@ nginxGateway:
     ## Some environments may need this set to true in order for the control plane to successfully reload NGINX.
     allowPrivilegeEscalation: false
 
+  productTelemetry:
```

Comment:
shall we generate separate installation manifests for when telemetry enabled and disabled?

Reply:
I'd like to avoid installation manifests for so many different features. We are inevitably going to have more things in the future that change the manifests. It's one of the benefits of using helm (configurability).